controllers/operands/kubevirt.go

```diff
@@ -183,6 +184,11 @@ type kubevirtHooks struct {
 	cache *kubevirtcorev1.KubeVirt
 }
 
+type rateLimits struct {
+	Qps   int
```

Comment:
will it work to change it to `float32`, to get rid of the casting below?

Reply:
Done.